addons/binding/org.openhab.binding.evohome/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,36 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Evohome Binding
+Bundle-SymbolicName: org.openhab.binding.evohome;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.3.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Export-Package: 
+ org.openhab.binding.evohome,
+ org.openhab.binding.evohome.handler
+Import-Package: 
+ com.google.common.collect,
```

Comment:
As you are using this only in a single line, could you completely remove this dependency? We are currently trying to get rid off the use of Guava.